### PR TITLE
Gateway: configure ocelot.json.

### PR DESCRIPTION
**Pull request by Skava600**

### CODE REVIEW

File: RecordHub.Gateway/Program.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+builder.Configuration.AddJsonFile("ocelot.json", optional: false, reloadOnChange: true)
+    .AddEnvironmentVariables();
+
+builder.Services.AddOcelot(builder.Configuration);
+builder.Services.AddSwaggerForOcelot(builder.Configuration);
+builder.Services.AddControllers();
+
+builder.Services.AddSwaggerGen();
+var app = builder.Build(); app.UseRouting();
+app.UseRouting();
```
Comment (Y45K0VL3V) [inline RecordHub.Gateway/Program.cs:19]: Double UseRouting. No two commands in one line.

File: RecordHub.Gateway/ocelot.json [hunk 2]
```diff
@@ -0,0 +1,291 @@
+  {
+    "Routes": [
+      // Identity service
```
Comment (Y45K0VL3V) [inline RecordHub.Gateway/ocelot.json:3]: Is this file should be removed? Does its place will be taken by ocelot.json files, that in Config/Ocelot?

**Review by Y45K0VL3V — CHANGES_REQUESTED** [review]

File: RecordHub.Gateway/ocelot.json (hunk 2 above)
Comment (Skava600) [reply RecordHub.Gateway/ocelot.json:3]: I wanted to implement merging of ocelot files that located in config/ocelot, but its not supporting swagger for ocelot, so i just keeped these files for the case if i would remove swagger an implement client app

Comment (Y45K0VL3V) [reply RecordHub.Gateway/ocelot.json:3]: Got it

**Review by Y45K0VL3V — APPROVED** [review]

**Pull request merged**